internal/lock/local.go

```diff
@@ -34,16 +34,24 @@ func (l *localLockFile) equals(other *localLockFile) bool {
 		l.DevboxVersion == other.DevboxVersion
 }
 
-func (l *localLockFile) IsUpToDate() (bool, error) {
-	newLock, err := forProject(l.project)
+func isLocalUpToDate(project devboxProject) (bool, error) {
+	filesystemLocal, err := readLocal(project)
+	if err != nil {
+		return false, err
+	}
+	newLock, err := forProject(project)
 	if err != nil {
 		return false, err
 	}
 
-	return l.equals(newLock), nil
+	return filesystemLocal.equals(newLock), nil
```

Comment:
this is confusing to read. Could we call them either:
`filesystemLock.equals(newLock)`
or
`filesystemLocal.equals(newLocal)`

since they are conceptually the same kind of object